### PR TITLE
Removes - unused methods and classess using Android Studio Tool - Part 2

### PR DESCRIPTION
**Pull request by AjeshRPai**
Fixes #

## Description 
This PR removes unused code from the Repo using Android Tool. `Code → Analyze → Run inspection by name..., type Unused declaration`

2nd Part of https://github.com/wordpress-mobile/WordPress-Android/pull/19788

Screenshot 2023-12-26 at 10 31 26 AM

## To Test:
1. Smoke test the JP And WP app and verify that everything is working as expected 
2. Verify that the classess, methods and files deleted are unused. 

## Regression Notes

1. Potential unintended areas of impact
App is not working as expected

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and Smoke testing 

4. What automated tests I added (or what prevented me from doing so)
Nothing

PR submission checklist:
- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

UI Changes testing checklist:
- [x] Portrait and landscape orientations.
- [x] Light and dark modes.
- [x] Fonts: Larger, smaller and bold text.
- [x] High contrast.
- [x] Talkback.
- [x] Languages with large words or with letters/accents not frequently used in English.
- [x] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [x] Large and small screen sizes. (Tablet and smaller phones)
- [x] Multi-tasking: Split screen and Pop-up view. (Android 10 or higher)

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsUtils.java [hunk 1]
```diff
@@ -641,13 +641,6 @@ public static void trackAnalyticsAccountCreated(String username, String email, M
         AnalyticsTracker.track(AnalyticsTracker.Stat.CREATED_ACCOUNT, properties);
     }
 
-    private static Map<String, String> createNewsCardProperties(String origin, int version) {
-        Map<String, String> properties = new HashMap<>();
-        properties.put(NEWS_CARD_ORIGIN, origin);
-        properties.put(NEWS_CARD_VERSION, String.valueOf(version));
```
Comment (irfano) [inline WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsUtils.java:647]: We can remove also `AnalyticsUtil.NEWS_CARD_ORIGIN` and `AnalyticsUtil.NEWS_CARD_VERSION`.

File: WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java
Comment (irfano) [inline WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java:1]: We can also remove the definition of these keys, `DeletablePrefKey.STATS_WIDGET_KEYS_BLOGS`, `DeletablePrefKey.STATS_WIDGET_DATA`, `DeletablePrefKey.STATS_WIDGET_DATA`, `DeletablePrefKey.STATS_WIDGET_PROMO_ANALYTICS`, `UndeletablePrefKey.IAP_SYNC_REQUIRED`, `DeletablePrefKey.PENDING_DRAFTS_NOTIFICATION_LAST_NOTIFICATION_DATES`.

**Review by irfano — COMMENTED** [review]
I found out some unused fields while reviewing. Then I noticed this PR only covers methods and classes and I refrained from delving into the details to find more unused fields.
When I run unused declaration for only methods and classes, there are still 317 warnings. Some of these warnings are false alarms, but others are really unused. Did you intentionally leave them in the code?  Here are few examples:
- `AztecEditorFragment.canUndo()`
- `AztecEditorFragment.MediaPredicate.getTempMediaIdPredicate()`
- `AppPrefs.getReaderRecommendedTagsDeletedForLoggedOutUser()`
- `LocaleManager.createLanguageDetailDisplayStrings()`

Was your intention to remove all unused methods and classes? If that is the case, I can go over all warnings to find unused declarations that are really unused.
I have smoke-tested the basic features, and they are still working properly 🟢. I believe this PR is safe. We can either add more unused code and merge this, or merge this and perform other cleanups in a separate PR.

**Comment by AjeshRPai** [comment]
Hey @irfano 

Thanks for the review. 🙌🏼 

>When I run unused declaration for only methods and classes, there are still 317 warnings. Some of these warnings are false alarms, but others are unused. Did you intentionally leave them in the code? Here are a few examples

I reverted some commits, which deleted the unused functions in the Editor libs. I intend to take it up removing of the unused methods from Editor libs in a separate PR. 


> Was your intention to remove all unused methods and classes? If that is the case, I can go over all warnings to find unused declarations that are really unused.

Yup, I was concentrating more on Classes and Methods. There is a chance that I might have missed some methods or classes. If you can, go over all the warnings that are still there. It would be really helpful. I suggest you do it in a separate PR if possible.  Otherwise, I can take it up and create a Part 3 PR.

**Review by irfano — APPROVED** [review]
We discussed with Ajesh and decided to continue removing the remaining warnings in other PRs.
The changes in this PR LGTM! 👍🏻 Merging now.

**Pull request merged**